src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -36,6 +36,9 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
         url,
         params: {
           name: encodeURIComponent(file.name)
+        },
+        headers: {
+          Authorization: 'Basic '+localStorage.getItem('authorization_token')
```

Comment:
This concatenation doesn't work! Or the app hasn't been deployed after adding this row.

Reply:
You're right I forgot to deploy it.